flink-connectors/flink-connector-elasticsearch6/pom.xml

```diff
@@ -195,7 +195,6 @@ under the License.
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-surefire-plugin</artifactId>
-				<version>2.12.2</version>
```

Comment:
Without this fix, the testing for this module fails when `groups` is specified in maven building. The error is as below:

> initializationError(org.junit.runner.manipulation.Filter)  Time elapsed: 0.002 sec  <<< ERROR!
> java.lang.Exception: No tests found matching *org.apache.flink.testutils.junit.category.AlsoRunWithSchedulerNG from org.junit.runner.Request$1@7e0ea639
> 	at org.junit.internal.requests.FilterRequest.getRunner(FilterRequest.java:40)
> 	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
> 	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:62)
> 	at org.apache.maven.surefire.junitcore.JUnitCoreProvider.invoke(JUnitCoreProvider.java:139)
> 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> 	at java.lang.reflect.Method.invoke(Method.java:498)
> 	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
> 	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
> 	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
> 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:113)
> 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)